lib/compat/wordpress-6.2/get-global-styles-and-settings.php

```diff
@@ -48,6 +48,35 @@ function wp_theme_has_theme_json() {
 	}
 }
 
+if ( ! function_exists( 'wp_theme_get_css_selector_for_block' ) ) {
+	/**
+	 * Lookup a CSS selector for the block provided, and return it if it exists.
+	 *
+	 * @param string $block_name The name of the block to lookup the CSS selector for.
+	 *
+	 * @return string|null the CSS selector for the block.
+	 */
+	function wp_theme_get_css_selector_for_block( $block_name ) {
+		$registry = WP_Block_Type_Registry::get_instance();
+		$blocks   = $registry->get_all_registered();
```

Comment:
If the method is about a single block, we can use `get_registered( $name )` instead.